### PR TITLE
Resolve a crash when EyeToy is opened the second time

### PR DESCRIPTION
**Pull request by Florin9doi**

### CODE REVIEW

File: pcsx2/USB/usb-eyetoy/cam-windows.cpp [hunk 1]
```diff
@@ -455,6 +455,7 @@ namespace usb_eyetoy
 				samplegrabberfilter->Release();
 				samplegrabber->Release();
 				nullrenderer->Release();
+				sourcefilter = NULL;
```
Comment (lightningterror) [inline pcsx2/USB/usb-eyetoy/cam-windows.cpp:458]: Use nullptr instead?

**Review by lightningterror — APPROVED** [review]

**Comment by lightningterror** [comment]
What are the steps to reproduce the issue?

**Comment by Florin9doi** [comment]
Have a webcam connected and configured in usb configuration page and then boot EyeToy Kinetic twice

**Comment by Florin9doi** [comment]
2 weeks to null a pointer ?

**Comment by refractionpcsx2** [comment]
@jackun do you want to deal with this?

**Pull request merged**